src/allencell_ml_segmenter/core/prediction_result_input_widget.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Optional
+
+from allencell_ml_segmenter.core.file_input_model import (
+    InputMode,
+    FileInputModel,
+)
+from allencell_ml_segmenter.core.file_input_widget import FileInputWidget
+from allencell_ml_segmenter.main.i_viewer import IViewer
+from allencell_ml_segmenter.main.segmenter_layer import LabelsLayer
+from qtpy.QtCore import Qt
+
+from allencell_ml_segmenter.prediction.service import ModelFileService
+
+from napari.utils.events import Event as NapariEvent  # type: ignore
+
+
+class PredictionResultListWidget(FileInputWidget):
+    """
+    Widget containing a list of prediction results that are selectable for thresholding
+    """
+
+    def __init__(
+        self, model: FileInputModel, viewer: IViewer, service: ModelFileService
+    ):
+        super().__init__(
+            model, viewer, service, include_channel_selection=False
```

Comment:
what functionality of the superclass does `PredictionResultListWidget` use?  My concern is that these args are 'just along for the ride'.